working/m12-strings-english.txt

```diff
@@ -1104,216 +1104,216 @@
-^L1300^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<...Please give us strength,[22 FF][01 FF]@if it is possible...[01 FF]  Please...>[22 FF][01 FF]@<................[01 FF]  Somebody... help us...>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1301^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<...Please give us strength![22 FF][01 FF]@Anyone who can hear our plea...[01 FF]  Help us...>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1302^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<...Please give us strength![22 FF][01 FF]@Speed this prayer[01 FF]  to all the people of the earth.>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1303^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<...Please give us strength![22 FF][01 FF]@Anyone who can hear our plea...[01 FF]  Help us...>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1304^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<...Please grant us power![22 FF][01 FF]@Anyone who can hear our plea...[01 FF]  Help us...>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1305^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<...Please grant us strength![22 FF][01 FF]@Anyone who can hear our plea...[01 FF]  We ask for this...>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1306^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<...Please grant us power![22 FF][01 FF]@Speed this prayer[01 FF]  to all the people of the earth...>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1307^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<I can't think of[01 FF]  anyone else...[22 FF][01 FF]@...Someone, anyone...[01 FF]  ...please help us.>[61 FF 01 00][1B FF 96 00][C6 FF][00 FF]
-^L1308^[D4 FF 1C 00]@[9F FF] prayed[01 FF]  from the bottom of her heart![02 FF][1B FF 3C 00][61 FF 02 00]@<Someone... can you hear me?![22 FF][01 FF]@Please, give us strength!>[61 FF 01 00][1B FF 96 00][00 FF]
-^L1309^[86 FF _L1311_][1B FF 28 00][68 FF]@You cannot grasp the true form[01 FF]  of [9F FF]' attack![F0 FF 31 00 00 00][1B FF 78 00][F0 FF 36 00 00 00][D4 FF 51 00][1B FF 28 00][D4 FF 51 00][00 FF]
-^L1310^[86 FF _L1311_][1B FF 28 00][68 FF][86 FF _L1311_][1B FF 28 00][68 FF][86 FF _L1311_][1B FF 28 00][68 FF]@You cannot grasp the true form[01 FF]  of [9F FF]' attack![F0 FF 31 00 00 00][1B FF 78 00][F0 FF 36 00 00 00][D4 FF 51 00][1B FF 28 00][D4 FF 51 00][00 FF]
+^L1299^@All of a sudden,[02 FF]@[86 FF _userlowerthe_][9F FF] gave off[01 FF]  a rainbow of colors![D4 FF 41 00][F0 FF 28 00 00 00][1B FF 1E 00][D4 FF 41 00][F0 FF 28 00 00 00][1B FF 1E 00][D4 FF 41 00][F0 FF 28 00 00 00][1B FF 1E 00][00 FF]
```

Comment:
Nitpick: it's not good in terms of grammar, but Earthbound's original logic capitalizes "the" after a bullet point.